docs/readmes/proposals/p009_nms_mariadb_migration.md

```diff
@@ -0,0 +1,160 @@
+
+1. Migration of NMS db from MariaDB to to Postgres to share with orc8r
+2. Migration Process
+
+## Goals
+
+Currently the Magma NMS and orc8r are running on separate database
+technologies. The orc8r uses postgres for model storage, whereas the NMS uses
+MariaDB for storage of users and organizations.
+
+There are no particular benefits for NMS to use MariaDB over Postgres
+(or vice-versa), and the continued usage of MariaDB incurs additional costs
+for network operators.
+Particularly, for network operators using Magma on AWS, there are operating
+costs to running both MariaDB and Postgres, over using just one or the other.
+The goal of this migration is to have both NMS and orc8r use the same DB 
+technology to reduce operating expenses of running a network on Magma.
+
+While orc8r does also use other technologies such as Prometheus and
+Elasticsearch, this document does not concern costs associated to such
+technologies.
+
+## Success Criteria
+
+Several success criteria 
+
+- Migration script only required to run once after upgrade to 1.5
+- Networks running on 1.5 do not have expenses related to running MariaDB
+- Migration process works in the three scenarios:
+    - with bare Docker only
+    - with Kubernetes only
+    - with Terraform and Kubernetes
+    
+## Current Setup
+
+#### NMS Storage
+
+The NMS for the most part only consumes the orc8r REST API, and few data
+objects are interacted with without using the orc8r API.
+This data includes only the following:
+- Organizations
+- NMS Users
+- NMS Feature Flags
+- NMS Audit Log Entries
+
+To store this NMS-native data in MariaDB, the [Sequelize](https://sequelize.org/master/)
+ORM is used. Sequelize is configured through environment variables to use
+MariaDB, though Sequelize is DB agnostic, and can be configured to use
+Postgres.
+
+#### Terraform
+
+Terraform specifies a single AWS DB instance for Postgres, and another AWS
+DB instance for MariaDB on NMS.
+
+## Proposed End Goal Setup
+
+#### NMS Storage
+
+For NMS-native data, Sequelize will still be used, but will be configured to
+use Postgres. A separate logical Postgres DB will be used for namespacing
+purposes between orc8r data and NMS data.
+
+#### Terraform
+
+A single AWS DB instance will be specified for Postgres, with two logical DBs.
+
+References:
+- [Terraform: AWS DB Resource Specification](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance)
+- [AWS FAQ: How many databases or schemas can I run within a DB instance?](https://aws.amazon.com/rds/faqs/)
+- [StackOverflow: Using Terraform for multiple logical DBs](https://stackoverflow.com/questions/52542244/provision-multiple-logical-databases-with-terraform-on-aws-rds-cluster-instance)
+
+
+## Migration Process
+
+This section is split up between the process that operators must follow to
+successfully migrate NMS off MariaDB, and the technical details to achieve
+this migration.
+
+#### Operator Process - Docker 
```

Comment:
Since the docker is a developer version we don't need to provide a migration script here. We can just leave it as is.